src/whisper_transcriber/metadata_formatter.py

```
@@ -0,0 +1,145 @@
+        duration = metadata.get('duration', 0)
+        if duration:
+            lines.append(f"Duration: {format_time(duration)}")
+        
+        # Add upload date if available
+        upload_date = metadata.get('upload_date', '')
+        if upload_date:
+            try:
+                date_obj = datetime.strptime(upload_date, '%Y%m%d')
+                formatted_date = date_obj.strftime('%B %d, %Y')
+                lines.append(f"Upload Date: {formatted_date}")
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)